### PR TITLE
Added Virtual Mouse and Click Me game

### PR DESCRIPTION
**Pull request by toha87**
Check out the game :)

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -110,17 +110,17 @@ const App = () => {
             <LoggingContext.Provider value={{logger}}>
                 <AppContext.Provider value={{mongoHook, debuggingTools}}>
                     <VoiceCommandsContext.Provider value={{SpeechRecognitionHook}}>
-                        <Row>
-                            <Col>
```
Comment (toha87) [inline src/App.js]: Let me know if this was for a specific reason. It was causing a weird behavior in my Click Me game

File: src/hooks/useHandGestures.js [hunk 2]
```diff
@@ -52,8 +55,15 @@ export const useHandGestures = () => {
...
                 predictions.map((item) => {
+
+                    let xLocal = item.bbox[0] + (item.bbox[2] / 2);
+                    let yLocal = item.bbox[1] + (item.bbox[3] / 2);
+
+                    mouseClick(xLocal, yLocal);
+
                     setX(item.bbox[0] + (item.bbox[2] / 2));
                     setY(item.bbox[1] + (item.bbox[3] / 2));
+
```
Comment (blazinaj) [inline src/hooks/useHandGestures.js:66]: Very cool!

File: src/App.js [hunk 3]
```diff
@@ -110,17 +110,17 @@ const App = () => {
...
+                        <>
+                            <>
                                 <Routing/>
-                            </Col>
+                            </>
                             {
                                 showLogger &&
-                                <Col>
+                                <>
```
Comment (blazinaj) [inline src/App.js:120]: Does the logger still display properly?

File: src/hooks/useHandGestures.js [hunk 4]
```diff
@@ -108,23 +118,106 @@ export const useHandGestures = () => {
...
+            <br/>
+            <h2 style={{color: "white"}}>Score: {' '}
+                <span id="score" style={{color: "yellow"}}>0</span></h2>
+            <br/>
+
+            <Button color="primary" id="increaseScoreButton" onClick={() => clicked()}
+                    style={{zIndex: 1, position: "absolute", top: "50%", left: "50%"}}>Click
+                Me!</Button>
+
+            <canvas id="draw-canvas" style={{backgroundColor: "black", display: "none"}}></canvas>
```
Comment (blazinaj) [inline src/hooks/useHandGestures.js:214]: Sweet!

**Review by blazinaj — APPROVED** [review]
This is awesome, I'm excited to play it

File: src/hooks/useHandGestures.js [hunk 5]
```diff
@@ -108,23 +118,106 @@ export const useHandGestures = () => {
...
+            "click",
+            true /* bubble */, true /* cancelable */,
+            window, null,
+            x, y, 0, 0, /* coordinates */
+            false, false, false, false, /* modifier keys */
+            0 /*left*/, null
+        );
+        el.dispatchEvent(ev);
+    };
+
```
Comment (NikolasGraham) [inline src/hooks/useHandGestures.js:134]: This is neat, I'm excited to try this out!

**Review by NikolasGraham — APPROVED** [review]

**Comment by temichpilot** [comment]
Nice! Well done!

**Review by temichpilot — APPROVED** [review]
Good job Anatoly!

**Review by JeffHowes — COMMENTED** [review]
I played the click me game, logged out, logged in as guest, when to Paint page and got the following error.  Have you seen this bug before?
image

**Comment by toha87** [comment]
> I played the click me game, logged out, logged in as guest, when to Paint page and got the following error. Have you seen this bug before?
> image

Thanks Jeff:) good QA

**Pull request merged**